cmd/utils/flags_legacy.go

```diff
@@ -74,7 +74,13 @@ func showDeprecated(*cli.Context) {
 	fmt.Println("The following flags are deprecated and will be removed in the future!")
 	fmt.Println("--------------------------------------------------------------------")
 	fmt.Println()
-	// TODO remove when there are newly deprecated flags
-	fmt.Println("no deprecated flags to show at this time")
+	// These are the deprecated aliased flags
+	fmt.Println("--nousb")
```

Comment:
A better way to do this is to have it like it was earlier, see https://github.com/ethereum/go-ethereum/commit/8e547eecd592fe3306e39a4fea703dc1307b8651#diff-3901b608e5eb7baea94e1787658551873145235ef2b564350ca3ebb9dee2736aL183-L185

Reply:
Bundle up all of them in `DeprecatedFlags` (if they aren't already bundled), and then do
```
for _, flag := range DeprecatedFlags {
		fmt.Println(flag.String())
	}
```

Reply:
Done!